symex-interop.el

```diff
@@ -89,7 +92,9 @@ right symex when we enter Symex mode."
         ((symex--evil-enabled-p)
          (evil-normal-state))
         ((symex--evil-installed-p)
-         (evil-emacs-state))))
+         (evil-emacs-state))
+        ((fboundp 'symex-user-defined-higher-mode)
+         (symex-user-defined-higher-mode))))
```

Comment:
@countvajhula  I'm not sure if there a is a better way or a standard way of exposing user functions.

Reply:
I think we can just use `symex-meow` here rather than expose it to an arbitrary user customization.